src/main/java/firebase/sdk/dashboard/data/SDKReleaseMetadata.java

```diff
@@ -39,6 +41,8 @@ public static Builder newBuilder() {
 
     Builder releaseName(String releaseName);
 
+    Builder verifier(String verifier);
```

Comment:
Who is the verifier?

Reply:
I believe the verifier is the person who enrolled in the release. It is in the figma prototype. Will add a comment to clarify